packages/sw-lib/src/lib/sw-lib.js

```diff
@@ -351,6 +352,25 @@ class SWLib {
       requestWrapper: new RequestWrapper(wrapperOptions),
     });
   }
+
+  /**
+   * This method will register any default routes the library will need.
+   */
+  _registerDefaultRoutes() {
+    const revisionedManager = this._precacheManager.getRevisionedCacheManager();
+    const cacheFirstHandler = this.cacheFirst({
+      cacheName: revisionedManager.getCacheName(),
+    });
+
+    const route = new this.Route({
+      match: ({url, event}) => {
+        const cachedUrls = revisionedManager.getCachedUrls();
```

Comment:
I am  more than happy to do that, only edge case (and it is an edge case) is to reset that variable is more entries are added, but I'll go with this approach because I think it's a little safer / easier to follow.